src/components/maps/nite-overlay.js

```diff
@@ -114,6 +120,72 @@ const nite = {
 
     return new google.maps.LatLng(lat, lng)
   },
+  // Don't ask me how this works, formula from here -> https://stjarnhimlen.se/comp/tutorial.html#7
+  calculatePositionOfMoon: function (date) {
+    date = new Date()
+    // Get the Julian Day Number for the given date
+    const jdn = (date.getTime() / 86400000) + 2440587.5; // Convert milliseconds to days
+
+    // Reference epoch (J2000)
+    const referenceJDN = 2451545.0;
+
+    // Calculate d
+    const d = jdn - referenceJDN;
+
+    // Constants
+    const degToRad = Math.PI / 180;
+    const radToDeg = 180 / Math.PI;
+
+    // Normalize angle to 0-360 degrees
+    function rev(angle) {
+      const normalizedAngle = angle % 360;
+      return normalizedAngle >= 0 ? normalizedAngle : normalizedAngle + 360;
+    }
+
+    // Initial orbital elements
+    const N = rev(125.1228 - 0.0529538083 * d);
+    const i = 5.1454;
+    const w = rev(318.0634 + 0.1643573223 * d);
+    const a = 60.2666;
+    const e = 0.054900;
+    const M = rev(115.3654 + 13.0649929509 * d);
+
+    // Normalize M
+    const normalizedM = rev(M + 129 * 360);
```

Comment:
I'm not sure this line is necessary.

Reply:
Looking at that now, yeah that doesn't make any sense. Theres a line in the article that says 'To normalize M we had to add 129*360 = 46440 degrees.' But if it outputs the same value then I will remove it

Reply:
I think they were just giving an example of what the normalization does, rather than adding an extra step